README.md

```diff
@@ -1,54 +1,119 @@
-<picture>
-  <source media="(prefers-color-scheme: dark)" srcset="./images/gradle-white-primary.png" width="350px">
-  <img alt="Gradle" src="./images/gradle-dark-green-primary.png" width="350px">
-</picture>
+<div align="center" style="margin-bottom: 30px;" >
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="images/gradle-dark-green-primary.png" width="700px">
+    <img alt="Gradle Logo" src="images/gradle-white-primary.png" width="600px">
+  </picture>
+</div>
 
-[![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.gradle.org/scans)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/4898/badge)](https://bestpractices.coreinfrastructure.org/projects/4898)
+<div align="center" style="margin-top: 20px;">
 
-[Gradle](https://gradle.org/) is a build tool with a focus on build automation and support for multi-language development. If you are building, testing, publishing, and deploying software on any platform, Gradle offers a flexible model that can support the entire development lifecycle from compiling and packaging code to publishing websites. Gradle has been designed to support build automation across multiple languages and platforms, including Java, Scala, Android, Kotlin, C/C++, and Groovy, and is closely integrated with development tools and continuous integration servers, including Eclipse, IntelliJ, and Jenkins.
+[![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.gradle.org/scans) <br>
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/4898/badge)](https://bestpractices.coreinfrastructure.org/projects/4898) <br>
+[![Join the Gradle Community Slack](https://img.shields.io/badge/Slack-Join%20the%20Community-brightgreen?style=flat&logo=slack)](https://gradle.org/slack-invite) <br>
+[![Gradle Documentation](https://img.shields.io/badge/Docs-Gradle%20User%20Guide-blueviolet?style=flat&logo=read-the-docs)](https://docs.gradle.org/current/userguide/userguide.html) 
```

Comment:
I get this render in the dark mode 
![Screenshot 2024-10-29 at 19 41 05](https://github.com/user-attachments/assets/f52d6f9f-c1da-44e7-9047-4c5acfea60a0)

Line breaks between labels are not helpful, and the logo

Reply:
Okay! I have removed those line breaks.